### PR TITLE
 build: Add MSVC project files for bitcoin-wallet-tool

### PR DESCRIPTION
**Pull request by ken2812221**
Based on #13926. Add msvc project files for it

### CODE REVIEW

File: src/wallet/wallettool.cpp [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            return false;
+        }
+
+        CWallet dummyWallet("dummy", WalletDatabase::CreateDummy());
+        std::string backup_filename;
+        if (!WalletBatch::Recover(path, (void*)&dummyWallet, WalletBatch::RecoverKeysOnlyFilter, backup_filename)) {
+            fprintf(stderr, "Salvage failed\n");
+            return false;
+        }
+        //TODO, set wallets best block to genesis to enforce a rescan
```
Comment (practicalswift) [inline src/wallet/wallettool.cpp]:
```
2018-09-22 20:39:03 cpplint(pr=14284): src/wallet/wallettool.cpp:125:  Should have a space between // and comment  [whitespace/comments
```

Comment (ken2812221) [reply src/wallet/wallettool.cpp]: This is not my commit, please comment on #13926.

**Comment by ken2812221** [comment]
Rebased

**Comment by jnewbery** [comment]
Thanks for doing this @ken2812221 . I've squashed your changes into the commit in #13926. (I didn't cherry pick as a separate commit to avoid breaking git bisect for msvc builds).

**Pull request closed without merging**